src/pages/about.js

```diff
@@ -0,0 +1,191 @@
+
+              <List component="nav">
+                {dynamicUsers.nodes.map((contributor) => {
+                  const displayName = contributor.name?? contributor.login;
+                  return (
+                    <ListItem button component="a" href={contributor.htmlUrl} key={contributor.login}>
+                      <ListItemAvatar>
+                        <Avatar alt={displayName} src={contributor.avatarUrl} />
+                      </ListItemAvatar>
+                      <ListItemText primary={displayName}/>
+                    </ListItem>
+                  )
+                })}
+              </List>
+
+            </CardContent>
+          </Card>
+
+          <Card>
+            <CardContent>
+              <Typography variant="h5" component="h2">Funding</Typography>
+
+              <div className={classes.enrtf}>
+
+                <CardMedia
+                  className={classes.media}
+                  component={Img}
+                  fixed={enrtfLogo.childImageSharp.fixed}
+                  alt="ENRTF Logo"
+                  title="ENRTF Logo"
+                />
+
+                <div className={classes.enrtfText}>
+                  <Typography paragraph>
+                    Funding for this project was provided by the Minnesota Environment and Natural Resources Trust Fund as recommended
+                    by the <a href="https://www.lccmr.leg.mn/">Legislative-Citizen Commission on Minnesota Resources (LCCMR)</a>
+                  </Typography>
+                  <Typography>
+                    The Trust Fund is a permanent fund constitutionally established by the citizens of Minnesota to assist in the
+                    protection, conservation, preservation, and enhancement of the state’s air, water, land, fish, wildlife, and
+                    other natural resources. Currently 40% of net Minnesota State Lottery proceeds are dedicated to growing
+                    the Trust Fund and ensuring future benefits for Minnesota’s environment and natural resources.
+                  </Typography>
+                </div>
+
+              </div>
+
+            </CardContent>
+          </Card>
+
+          <Card>
+            <CardContent>
+              <Typography gutterBottom variant="h5" component="h2">License</Typography>
+              <Typography>
+                The images used for facts belong to their respective owners. Clicking the info button on the bottom right of a fact
+                presents the information and links to the image owners as well as the license under which the image is published.
+              </Typography>
+            </CardContent>
+          </Card>
+
+        </div>
+      </div>
+
+    </Layout>
+  )
+}
+
+export default About;
+
+export const aboutQuery = graphql`
+query {
+  enrtfLogo: file(relativePath: { eq: "enrtf_logo.png" }) {
+    childImageSharp {
+			fixed(width: 180) {
+				...GatsbyImageSharpFixed_withWebp
```

Comment:
The blur effect really doesn't work well on the logo so I think we should do the same as on the footer and disable it.

```suggestion
				...GatsbyImageSharpFixed_withWebp_noBase64
```